### DISCUSSION TITLE
Hide the save button from Update UI

### DISCUSSION THREAD

**Discussion by aberaldo** [Laravel-Backpack/community-forum#276] [General] [answered]
Hello, im wondering if its posible to use the update operation UI with all fields but hidding the save action button. I want to use the update UI only for showing the fields. Its possible??

Like this
![image](https://user-images.githubusercontent.com/8792209/139261097-30ff1626-9a39-4228-abf9-6be550bd71e2.png)

Any help?

Thanks.

**Comment by pxpm** [comment]
Hello @aberaldo 

You can achieve a similar result setting up in your `setupCreateOperation`: 

```php
$this->crud->replaceSaveActions([
            'name' => 'Return',
            'redirect' => function($crud,$request,$id) {
                return redirect()->back();
            }
        ]);
        $this->crud->setOperationSetting('showCancelButton', false);
```

**Reply by aberaldo** [reply]
Good! But when the redirect is executed a message appears like as the model was edited

![image](https://user-images.githubusercontent.com/8792209/139461137-bb38af7a-bc65-441f-aacd-144373aa8cb4.png)

Any suggestions?

**Comment by aberaldo** [comment]
Is there a solution to this? Its seems that te edit action is executed. I want to hide that button.
Thx

**Comment by aberaldo** [comment]
If i hide all actions exception appears:
![image](https://user-images.githubusercontent.com/8792209/140969341-f1798ff3-0e54-4e35-aaff-87ffa57c4f9e.png)

**Comment by aberaldo** [comment, marked as answer]
@pxpm  I solved hidding the button from the view with css and in the authorize method of the request always returning false.
As an improvement i think it should be an easy way to do this. Maybe allowing deleting all save actions without throwing an exception?
Thanks
